README.md

```diff
@@ -1,177 +1,37 @@
-[![Build Status](https://travis-ci.org/RedHatInsights/insights-frontend-starter-app.svg?branch=master)](https://travis-ci.org/RedHatInsights/insights-frontend-starter-app)
+[![Build Status](https://travis-ci.org/RedHatInsights/insights-frontend-components.svg?branch=master)](https://travis-ci.org/RedHatInsights/insights-chrome)
+[![npm version](https://badge.fury.io/js/%40red-hat-insights%2Finsights-frontend-components.svg)](https://badge.fury.io/js/%40red-hat-insights%2Finsights-frontend-components)
 
-# insights-frontend-starter-app
-React.js starter app for Red Hat Insights products that includes Patternfly 3 and Patternfly Next.
+# insights-frontend-components
+Red Hat Insights Components for applications in a React.js environment
 
-# Build app
-1. ```npm install```
+# Usage
+npm: `npm install @red-hat-insights/insights-frontend-components`
 
-2. ```npm run start```
-    - starts webpack bundler and serves the files with webpack dev server
+# Development
+Create a component inside of the PresentationalComponent folder
 
-### Testing
-- Travis is used to test the build for this code.
-    - `npm test` will run the test
-
-# Documentation links
- * Utils
-   * [Redux](doc/utils/redux.md)
```

Comment:
We need to document utils and maybe even components, how about creating new section with doc links to each util/component?

Reply:
Updated!